referral_module/api/tests/test_campaigns_api.py

```diff
@@ -0,0 +1,104 @@
+from django.test import TestCase
+
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+from referral_module.tests import factories
+from referral_module import models
+
+
+class CampaignsApiTestCase(TestCase):
+    def setUp(self):
+        # Every test needs a client.
+        self.user = User.objects.create_user('usertest', 'user@mail.com', 'liorlior')
+        self.client.login(username='usertest', password='liorlior')
+        self.campaign = []
+        for i in range(0, 5):
+            self.campaign.append(factories.CampaignFactory(
+                description='some some'
+            ))
+
+    def test_campaigns_list_api(self):
+        url = reverse(
+            'api-campaigns-list'
+        )
+        response = self.client.get(url)
+
+        # Check that the response is 200 OK.
+        self.assertEqual(response.status_code, 200)
+
+        data = response.json()
+        # Check that the rendered context contains 5 customers.
+        self.assertEqual(data['count'], 5)
+
+    def test_campaigns_details_api(self):
+        pk = self.campaign[0].id
```

Comment:
`self.campaign[0]` is used almost in every test method, would be nice to have it as a part of `setUp` method.